client/src/app/create/create.component.scss

```diff
@@ -0,0 +1,19 @@
+.panel {
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.panel {
+  &-body {
+    margin-bottom: 15px;
+  }
+
+  &-footer {
+
+    .list-group{
```

Comment:
myślę, że warto z początku poświęcić czas na refaktoring takich pierdół, pozwoli nam to później się lepiej zgrać.